build/Makefile

```diff
@@ -49,7 +49,7 @@ MINIKUBE_PROFILE ?= agones
 GO_BUILD_TAGS ?= none
 BUILDX_WINDOWS_BUILDER = windows-builder
 WINDOWS_DOCKER_PUSH_ARGS =
-WINDOWS_VERSIONS = ltsc2019 # 1909 (For GKE WINDOWS_SAC support)
+WINDOWS_VERSIONS = ltsc2019 # 20H2 (For GKE WINDOWS_SAC support)
```

Comment:
Wondering if we can add a comment/link here, about what these terms mean?

I'm not sure what the change to 20H2 means?

Reply:
1909 is going end of life soon. GKE now supports 20H2.

Reply:
I don't know what those words mean 🙂 can we add a comment to point to the relevant docs somewhere?

Is the comment of `20H2` mean that we need to change something eventually? I'd just like this comment to be explicitly clear.

Reply:
Is this the right link for details on what these terms mean?
https://cloud.google.com/kubernetes-engine/docs/concepts/node-images#windows-server

If so, can we add a comment above for reference this line for reference?

Reply:
Added some docs about versioning.

Reply:
Perfect! Thanks so much!